source/compiler/qsc_doc_gen/src/generate_docs/tests.rs

```diff
@@ -181,6 +181,7 @@ fn index_file_generation() {
 
         | Name | Description |
         |------|-------------|
+        | [Complex](xref:Qdk.Std.Core.Complex) | Represents a complex number by its real and imaginary components. The first element of the tuple is the real component, the second one - the imaginary component. |
```

Comment:
Note that if you change the summary of Complex as suggested in the earlier comment, this will also need to be updated.